### PR TITLE
ko-KR: Apply #1245, #1257 and fix a translation

### PR DESCRIPTION
**Pull request by telk5093**
Applying for issue(s):
- #1245
- #1257

### CODE REVIEW

File: data/language/ko-KR.txt [hunk 1]
```diff
@@ -3494,7 +3494,7 @@ STR_5153    :테마 수정...
 STR_5154    :하드웨어 디스플레이
 STR_5155    :완성되지 않은 트랙의 테스트 허용
 STR_5156    :{SMALLFONT}{BLACK}대부분의 놀이기구 종류에서 트랙을 완성하지 않아도 테스트를 할 수 있도록 허용합니다. 블록 섹션 모드에는 적용되지 않습니다
-STR_5157    :모든 가격 잠금 해제
+STR_5157    :<사용 안 함>
```
Comment (Gymnasiast) [inline data/language/ko-KR.txt]: `<removed string - do not use>` should be included as-is, i.e. in English.

Comment (telk5093) [reply data/language/ko-KR.txt]: Done

File: data/language/ko-KR.txt [hunk 2]
```diff
@@ -3494,7 +3494,7 @@ STR_5153    :테마 수정...
 STR_5154    :하드웨어 디스플레이
 STR_5155    :완성되지 않은 트랙의 테스트 허용
 STR_5156    :{SMALLFONT}{BLACK}대부분의 놀이기구 종류에서 트랙을 완성하지 않아도 테스트를 할 수 있도록 허용합니다. 블록 섹션 모드에는 적용되지 않습니다
-STR_5157    :<사용 안 함>
+STR_5157    :모든 가격 잠금 해제
```
Comment (Gymnasiast) [inline data/language/ko-KR.txt]: <removed string - do not use>

Comment (telk5093) [reply data/language/ko-KR.txt]: Done. Thanks for noticing

**Pull request merged**